src/main/java/com/box/sdk/BoxAPIConnection.java

```diff
@@ -283,7 +287,7 @@ public boolean needsRefresh() {
         boolean needsRefresh;
 
         this.refreshLock.readLock().lock();
-        if (this.expires == 0) {
+        if (this.expires < 0) {
```

Comment:
We set expires to 0 to force the API connection to restart.

- -1 --> "null", "not set"
- 0  --> "force refresh the next time we need to"
- n  --> "needs refresh in n milliseconds"